js/views/pageNavVw.js

```diff
@@ -630,6 +630,8 @@ module.exports = baseVw.extend({
   },
 
   mouseenterServerSubmenuTrigger: function(e) {
+    this.serverSubmenu.css('right', this.$('.popMenu-navBarSubMenu-pageNav').outerWidth());
```

Comment:
It would be much simpler to just change popMenu-navBarSubMenu from position: absolute to float:right.

I'll merge this and then submit a patch.